### DISCUSSION TITLE
Can't issue cert using metamask - Ropsten test network

### DISCUSSION THREAD

**Discussion by RP-OIS-SK** [OpenCerts/opencerts-website#654] [Q&A]
I have some problem in issue test cert using metamask - Ropsten test network recently.  This is for testing in new certifcate template.  I have tried using the admin.opencerts.io and newadmin.opencerts.io in both windows and mac platform using Chrome and Firefox.  All do not work.  It was working for the past few years.    

Refer to https://github.com/OpenCerts/admin-website/issues/368

**Comment by ChenRen93** [comment]
Hi @RP-OIS-SK , i have just issued a document through "newadmin.opencerts.io". Could you try again to see if it is working now?

**Comment by ChenRen93** [comment]
Good evening @RP-OIS-SK , I have just referred to the post u made previously and did some checks.

U are using a wallet address of "https://ropsten.etherscan.io/address/0xeFD02EC4899ed25ac6Ba38D20c13CC109be53C38", this wallet address isn't the creator or owner of the document store u are trying to issue from "https://ropsten.etherscan.io/address/0x061aFcFF60b90E0F633F8ef157e01BaB9b2FfDD3", this is the cause of the error and status code.

The document store is created by "https://ropsten.etherscan.io/address/0x43e6a0fc3a21ebcc986f04ab783946185bf074e3". Please do use this ether wallet to issue the cert or alternatively, u can create a new document store and modify the DNS under ur current account for the issuance.

The last screenshot is answered by the OC maintainer. it shows that u are issuing an issued hash. If u are using it for a business demo, u can rewrap the document again to have other unique hash which can be used for other issuance. U may also want to increase the gas fee at the Metamask confirmation popup to speed up the transaction.

@simboonlong keep you in the loop :)

**Reply by RP-OIS-SK** [reply]
Thank you for explanation and suggestion.
